data/tokens.json

```diff
@@ -1289,360 +1289,78 @@
       "type": "typography"
     }
   },
-  "alerts": {
-    "Alert": {
-      "gtmds_alert-title": {
+  "avatar": {
+    "XLarge": {
+      "gtmds_avatar-label": {
         "value": {
-          "fontFamily": "{gtmds_font-family-p}",
-          "fontWeight": "{gtmds_font-weight-500}",
+          "fontFamily": "{gtmds_font-family-h2}",
+          "fontWeight": "{gtmds_font-weight-700}",
           "lineHeight": "{gtmds_line-height-base}",
-          "fontSize": "{Paragraph Size.gtmds_font-size-lg}",
-          "letterSpacing": "{gtmds_letter-spacing-none}",
+          "fontSize": "{gtmds_avatar-label_font-size-xl}",
+          "letterSpacing": "{gtmds_letter-spacing-sm}",
           "paragraphSpacing": "{gtmds_paragraph-spacing-none}",
-          "textCase": "{gtmds_text-case-none}",
+          "textCase": "{gtmds_text-case-uppercase}",
           "textDecoration": "{gtmds_text-decoration-none}"
         },
         "type": "typography"
-      },
-      "gtmds_alert-message": {
+      }
+    },
+    "Large": {
+      "gtmds_avatar-label": {
+        "value": {
+          "fontFamily": "{gtmds_font-family-h4}",
+          "fontWeight": "{gtmds_font-weight-700}",
+          "lineHeight": "{gtmds_line-height-base}",
+          "fontSize": "{gtmds_avatar-label_font-size-lg}",
+          "letterSpacing": "{gtmds_letter-spacing-sm}",
+          "paragraphSpacing": "{gtmds_paragraph-spacing-none}",
+          "textCase": "{gtmds_text-case-uppercase}",
```

Comment:
Is the text case really uppercase? @StefanIvanov

Reply:
I think yes since it is the initials of the avatar if I am reading it correctly